src/index.js

```diff
@@ -62,7 +62,14 @@ export default function loader(content) {
     );
   }
 
+  // Memcache output files
+  const cache = {};
```

Comment:
Agreed, most benchmarks show set to be 2 / 3 times faster than the old school object route.

Reply:
I think we are at the point where we can start being a bit more picky about performance related choices

Reply:
Done